OpenRA.Mods.RA/Guard.cs

```diff
@@ -18,10 +18,22 @@
 namespace OpenRA.Mods.RA.Traits
 {
 	[Desc("The player can give this unit the order to follow and protect friendly units with the Guardable trait.")]
-	class GuardInfo : TraitInfo<Guard> { }
+	class GuardInfo : ITraitInfo
+	{
+		public readonly string Voice = "Move";
+
+		public object Create(ActorInitializer init) { return new Guard(this); }
+	}
 
 	class Guard : IResolveOrder, IOrderVoice
```

Comment:
Can we rename this to `Guards` ?
`Guard` sounds more like like an activity than a trait.